### DISCUSSION TITLE
Plugin configuration

### DISCUSSION THREAD

**Discussion by MazahirA** [HaylLtd/cordova-plugin-background-geolocation#161] [General]
I'm using the below configuration:

BackgroundGeolocation.configure({
    locationProvider: BackgroundGeolocation.DISTANCE_FILTER_PROVIDER, 
    desiredAccuracy: BackgroundGeolocation.HIGH_ACCURACY,
   stopOnTerminate:false,
    stationaryRadius: 2,  
    distanceFilter: 5, 
    notificationTitle: 'Background Tracking',
    notificationText: 'getting your current location',
    debug: true,
    interval: 5000, 
    fastestInterval: 5000, 
    activitiesInterval: 5000,
   startForeground: true,
    url:'http://myserver',
    postTemplate: {
    Latitude: '@latitude',
    Longitude: '@longitude',
	Provider: '@provider',
	Accuracy: '@accuracy'
  }
  });
  
  The expected behavior:
Necessary:
1-The plugin should always send data in foreground, background and on terminate.
2- It should update/send user's location after every 5000-milli seconds (desired), in evert state (FG, BG, Terminate).

Additional:
3- It should update/send user's location after an exit from stationery state.
4- The plugin service should start and send data when a user moves 2-meters beyond stationary radius.

Additionally, what are the minimum values we  can set for following:
1-stationaryRadius.
2-distanceFilter.
3-fastestInterval.
4-interval.

**Comment by HarelM** [comment]
It is unclear if this is a discussion or an issue report.
In any case, you'll need to debug the code yourself if you want it solved.
